addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1051,10 +1034,8 @@ options.registry.CoverProperties = options.Class.extend({
 
         // Update saving dataset
         this.$target[0].dataset.coverClass = this.$el.find('[data-cover-opt-name="size"] we-button.active').data('selectClass') || '';
-        this.$target[0].dataset.textSizeClass = this.$el.find('[data-cover-opt-name="text_size"] we-button.active').data('selectClass') || '';
         this.$target[0].dataset.textAlignClass = this.$el.find('[data-cover-opt-name="text_align"] we-button.active').data('selectClass') || '';
         this.$target[0].dataset.filterValue = this.$filterValueOpts.filter('.active').data('filterValue') || 0.0;
-        this.$target[0].dataset.filterColor = this.$filterColorOpts.filter('.active').data('filterColor') || '';
```

Comment:
Just to be sure: migration script for that kind of change or it's just ok for that style to change ?

Reply:
no equivalent until new custo done by sam refactoring to apply filter on image directly

Reply:
Yep, my concern was more for the `textSizeClass` which will never have an equivalent. But I am fine with just removing it and say to the customers that the size changed, I did not want this option when we merged it anyway :smile:

Reply:
I'd say it is ok if there is some very small change like the text size when the user change version.
Otherwise I guess we would need to have an entry in cover_properties like 'migration_class' + keep unused css if there is.
Or removing the entry from cover_properties but keeping the variable in xml template that would be dead code just for migration issues that someone would probably delete some day since unused elsewhere.